lib/reporter.js

```diff
@@ -358,6 +360,16 @@ class AllureReporter extends events.EventEmitter {
         test.addLabel('severity', severity)
     }
 
+    allureIssue ({ cid, issue }) {
```

Comment:
Please, update corresponding README section https://github.com/webdriverio-boneyard/wdio-allure-reporter#supported-allure-api with new methods

Reply:
oh, forgot it... will do